### ISSUE TITLE
Sendungen von Amazon werden nicht angezeigt

### ISSUE THREAD

**Issue by chris76e** [TA2k/ioBroker.parcel#46]
Seit einiger Zeit werden bei mir keine Amazon Sendungen mehr angezeigt, bzw. nach neustart vom Adapter sind sie da, aber nach der 2. Aktualisierung verschwinden die Sendungen, offene und gelieferte. 

DHL, Hermes und DPD funktionieren. 

Getestet mit Version 0.0.27 und 0.0.28.

**Comment by Grothesk242** [comment]
Hatte das heute auch. Insgesamt 3 Sendungen von Amazon zu verschiedenen Lieferterminen und andauernd fing der counter für die insgesamt in Auslieferung befindlichen Sendungen auf Null.

**Comment by AlRo75** [comment]
Es ist völlig egal ob eine oder mehrere Sendungen. Zuerst werden sie angezeigt, bei der nächsten Aktualisierung sind sie weg und dann kommen sie wieder. Da-weg-da-weg-da-weg...

Festgestellt habe ich folgendes. Der DP:
`parcel.0.amazon.json`
wird immer korrekt befüllt und enthällt immer die Daten.

ALLERDINGS der DP
`parcel.0.allProviderJson`
ist mal befüllt und mal nicht.

**Comment by TA2k** [comment]
Wie sieht denn der Log dazu aus steht das keine Amazon Orders gefunden wurden

**Comment by AlRo75** [comment]
Wenn die Sendungen "verschwinden" steht drin, dass keine Amazon Orders gefunden wurden. 10 Minuten (das ist das eingestellt Intervall für Abrufe) kommt die Meldung nicht und die Sendungen sind wieder da.

**Comment by TA2k** [comment]
probier mal die github version ob die sich besser verhält

**Comment by Grothesk242** [comment]
Nein, habe die LogIns immer noch. Ich kann nur nicht sagen, ob/wie sich der DP
parcel.0.allProviderJson
verhält, hab gerade nix an Sendungen offen. 

```
2023-07-15 00:20:17.792  - info: parcel.0 (661178) No Amazon orders found
2023-07-15 00:20:20.299  - info: parcel.0 (661178) Login to Amazon successful
2023-07-15 00:45:17.267  - info: parcel.0 (661178) No Amazon orders found
2023-07-15 00:45:19.560  - info: parcel.0 (661178) Login to Amazon successful
```

**Comment by AlRo75** [comment]
Nach der Installation waren alle aktuellen Sendungen nicht da. Habe danach den Adapter neu gestartet und die aktuellen Sendungen waren wieder sofort da.

Aktueller Stand: Die Sendungen werden angezeigt und verschwinden nicht mehr. Von daher kann ich sagen, dass deine letzte Änderung erfolgreich war. 👍 Sieht gut aus.
